test/celeritas/user/CaloTestBase.cc

```diff
@@ -0,0 +1,83 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/user/CaloTestBase.cc
+//---------------------------------------------------------------------------//
+#include "CaloTestBase.hh"
+
+#include <iostream>
+
+#include "celeritas/global/Stepper.hh"
+#include "celeritas/user/StepCollector.hh"
+
+#include "ExampleCalorimeters.hh"
+
+using std::cout;
+
+namespace celeritas
+{
+namespace test
+{
+//---------------------------------------------------------------------------//
+CaloTestBase::~CaloTestBase() = default;
+
+//---------------------------------------------------------------------------//
+/*!
+ * Construct example callback and step collector at setup time.
+ */
+void CaloTestBase::SetUp()
+{
+    example_calos_ = std::make_shared<ExampleCalorimeters>(
+        *this->geometry(), this->get_detector_names());
+
+    StepCollector::VecInterface interfaces = {example_calos_};
```

Comment:
Looks like all the `StepCollector` tests use a single callback -- would it be useful to have an example where multiple step interfaces are combined?

Reply:
Undoubtedly... I'll add one.